universal-application-tool-0.0.1/app/repository/AmazonS3Client.java

```diff
@@ -103,9 +109,27 @@ private void getTestObject() {
 
   private void connect() {
     String regionName = config.getString(AWS_S3_REGION);
+    String endpoint;
     region = Region.of(regionName);
     bucket = config.getString(AWS_S3_BUCKET);
+    String dev_env = config.getString(DEV_ENV);
+    URI endpointOverride;
+
+    endpoint = config.getString(AWS_LOCAL_ENDPOINT);
+    try {
+        if (dev_env.equals("1")) {
```

Comment:
Suggestion (blocking): would you leverage `play.Environment#isDev` instead of having a dedicated config for this?